### PR TITLE
Banner image and icon update

### PR DESCRIPTION
**Pull request by timovanasten**
Increased the resolution of the banner image and removed the circles that were never intended to be there (in the design they were placeholders for icons I was told by @svenpopping). I've replaced them with actual icons.

### Before 
before
### After
after

### CODE REVIEW

**Comment by TimvdLippe** [comment]
The first two commits should not be there.

File: src/lancie-home-page/lancie-info-blocks.html [hunk 1]
```diff
@@ -11,21 +12,10 @@
...
   }
 
-  .container > div::before {
-    min-width: 36px;
-    min-height: 36px;
-    border-radius: 50%;
-    margin: auto 16px auto 0;
-    border: 4px solid var(--secondary-color);
-    background: transparent;
-    content: "";
```
[the remark this answers is not in the document]
Comment (timovanasten) [reply src/lancie-home-page/lancie-info-blocks.html]: This used to be for the circles that I changed into icons

**Comment by elarb** [comment]
Looks good! What do you think about using the ticket icon (`local-activity`) instead of the euro icon?

screenshot from 2018-12-25 05-42-14

**Comment by timovanasten** [comment]
I like it, will do

File: src/lancie-home-page/lancie-info-blocks.html [hunk 2]
```diff
@@ -11,21 +13,10 @@
   .container {
     display: flex;
     flex-wrap: wrap;
-    max-width: 1032px;
```
Comment (martijnjanssen) [inline src/lancie-home-page/lancie-info-blocks.html]: Is it not strange that the blocks are now wider than the site content.

Comment (timovanasten) [reply src/lancie-home-page/lancie-info-blocks.html]: I don't think these blocks need to be the same width as the other site content. At least I didn't like the restriction of 1032px as shown below. It leaves too little spacing between the blocks in my opinion, which is unneeded if you have room on the sides.

schermafbeelding 2018-12-25 om 13 26 58

I do agree that some sort of width restriction is nice. I like the spacing on my monitor which is 1440px. How about that?

**Pull request merged**